server/controllers/userController.js

```diff
@@ -97,10 +97,41 @@ const register = async (req, res) => {
   }
 };
 
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    // check if user exists
+    const findUser = await findUserByEmail(email);
+
+    if (!findUser) {
+      return res.json({ error: 'Incorrect Email or Password' });
+    }
+    // Load hash, compare password
+    const hash = await findUser.password;
+    const pwMatch = await bcrypt.compare(password, hash);
+
+    if (!pwMatch) {
+      return res.json({ error: 'Incorrect Email or Password' });
```

Comment:
Consider error structure as object with a message property, and return a 400 level status code.